public/javascripts/SVLabel/src/SVLabel/Tracker.js

```diff
@@ -29,7 +29,13 @@ function Tracker () {
         var prefix = "LowLevelEvent_";
 
         // track all mouse related events
-        $(document).on('mousedown mouseup mouseover mouseout mousemove click contextmenu dblclick', function(e) {
+        $(document).on('mousedown mouseup mouseover mouseout mousemove click contextmenu dblclick', function(e, extra) {
+            if (extra) {
+                if (typeof extra.lowLevelLogging !== "undefined" && !extra.lowLevelLogging) { // {lowLevelLogging: false}
+                    return;
+                }
+            }
+
```

Comment:
Is there a reason you are doing
```
if (x) {
    if (!y && !z) {
        return;
     }
}
// do stuff
```

Instead of
```
if (!x || y || z) {
    // do stuff
}
```
?

Reply:
Yes. I use 2 if statements in-case we want to do more with the "extra" field later on and I use `return;`  because it avoids deep nesting and is much cleaner IMO.

Reply:
Cool, makes sense if we are considering adding to the list of params in the future. Sounds good.